Makefile

```diff
@@ -6,6 +6,7 @@ clean:
 build:
 	./ssl_setup.sh
 	docker-compose build
+	docker-compose pull
```

Comment:
Should it akso build the project

Reply:
@oleg-nenashev I put `mvn clean install` in `make clean`. WDYT?